packages/agoric-cli/src/lib/rpc.js

```diff
@@ -0,0 +1,172 @@
+// @ts-check
+/* global Buffer, fetch */
+
+/**
+ * @template K, V
+ * @typedef {[key: K, val: V]} Entry<K,V>
+ */
+
+// TODO get non-local from config file
+export const networks = {
+  local: { rpc: 'http://0.0.0.0:26657', chainId: 'agoric' },
+  xnet: { rpc: 'https://xnet.rpc.agoric.net:443', chainId: 'agoricxnet-13' },
+  ollinet: {
+    rpc: 'https://ollinet.rpc.agoric.net:443',
+    chainId: 'agoricollinet-21',
+  },
+};
+
+/**
+ * Petnames depend on names issued by VBANK.
+ */
+export const vBankPetName = {
+  purse: {
+    anchor: 'AUSD',
+    ist: 'Agoric stable local currency',
+  },
+};
+
+export const bigIntReplacer = (_key, val) =>
+  typeof val === 'bigint' ? Number(val) : val;
+
+export const makeVStorage = network => {
+  const getJSON = url => fetch(network.rpc + url).then(res => res.json());
```

Comment:
This use of `fetch` is ambient authority. In due course, we should clean it up.

> No modules should export powerful objects (for example, objects that close over `fs`).
 -- https://github.com/Agoric/agoric-sdk/wiki/OCap-Discipline